java/src/jmri/jmrix/mrc/MrcSystemConnectionMemo.java

```diff
@@ -195,7 +195,7 @@ public MrcClockControl getClockControl() {
     /*public MrcConsistManager  getConsistManager() { return consistManager; }*/
 
     protected ResourceBundle getActionModelResourceBundle() {
-        return ResourceBundle.getBundle("jmri.jmrix.mrc.MrcActionListBundle"); //IN18N
+        return ResourceBundle.getBundle("jmri.jmrix.mrc.MrcBundle"); //IN18N
```

Comment:
The ```// NOI18N``` comments indicate to the ```ant i18n-consistency-check``` target that the String literal is not to be translated (and should not be included in the list of strings needing translation).
The ```//IN18N``` comment is noise as far as any technical process is concerned.